rskj-core/src/main/java/co/rsk/peg/FederationSupport.java

```diff
@@ -233,4 +236,19 @@ private boolean shouldFederationBeActive(Federation federation) {
         long federationAge = executionBlock.getNumber() - federation.getCreationBlockNumber();
         return federationAge >= bridgeConstants.getFederationActivationAge(activations);
     }
+
+    /**
+     * Get the Genesis Federation from a List of Genesis Federation Public Keys
+     *
+     * @return Federation
+     */
+    private Federation getGenesisFederation() {
+        final List<BtcECKey> genesisFederationPublicKeys = bridgeConstants.getGenesisFederationPublicKeys();
+        // IMPORTANT: BTC, RSK and MST keys are the same.
+        // Change upon implementation of the <INSERT FORK NAME HERE> fork.
+        final List<FederationMember> federationMembers = FederationMember.getFederationMembersFromKeys(genesisFederationPublicKeys);
+        final Instant genesisFederationAddressCreatedAt = bridgeConstants.getGenesisFederationAddressCreatedAt();
+        final FederationArgs federationArgs = new FederationArgs(federationMembers, genesisFederationAddressCreatedAt, 1L, bridgeConstants.getBtcParams());
```

Comment:
Let's put `1L` value in a variable, reading the code it's hard to understand what it represents

Reply:
Let me put it in a Constant and set a name that makes sense in the business logic.

Reply:
Done!